openpype/hosts/houdini/api/lib.py

```diff
@@ -836,6 +836,70 @@ def get_current_context_template_data_with_asset_data():
     return template_data
 
 
+def get_houdini_color_settings():
+    """Get Houdini working file color settings.
+
+    Returns:
+        Dict: The dictionary contains the Houdini working file color settings
+              if the settings are enabled, otherwise it is an empty dictionary.
+    """
+
+    project_settings = get_current_project_settings()
+    color_settings = project_settings["houdini"]["imageio"]["workfile"]
+    if color_settings["enabled"]:
+        color_settings.pop("enabled")
+        # Remove leading, and trailing whitespaces
+        view_Space = color_settings["review_color_space"]
+        color_settings["review_color_space"] = view_Space.strip()
```

Comment:
Are you storing a variable for the line lenght? Otherwise it could be:
```py
color_settings["review_color_space"] = color_settings["review_color_space"].strip()
```